content/capture-app/app-android-settings-configuration.md

```diff
@@ -749,6 +818,329 @@ In the same way, Event visualizations are considered valid if:
+
+Extra Type
+:   The data type of the returned value. Available options:
+    
+    - **String**: Plain text value
+    - **Integer**: Whole number value
+    - **Float**: Decimal number value
+    - **Boolean**: True/false value
+    - **JSON Object**: A single JSON object
+    - **List of JSON Objects**: An array of JSON objects
+
+Property in JSON Object
+:   *Only applicable when Extra Type is "JSON Object" or "List of JSON Objects"*
+    
+    Specifies the property name to extract from the JSON object(s). This should be a direct property of the JSON object.
+    
+    Example: `value`, `latitude`, `name`
+
+> **Note**
+>
+> - Only direct properties are supported. Nested properties (e.g., `location.coordinates.latitude`) are not currently supported.
+> - Array indexing (e.g., `results[0].name`) is not currently supported.
+> - When using **List of JSON Objects** as the Extra Type:
+>   - For **SEARCH**: The specified property will be extracted from all objects in the list, allowing users to search for tracked entity instances that match any value in the list.
+>   - For **DATA_ENTRY**: The values will be extracted from all objects and concatenated into a single string separated by commas (e.g., `value1,value2,value3`).
+
+![](resources/images/capture-app-image362.png)
+
+### Use Cases and Examples { #capture_app_android_settings_webapp_custom_intents_examples }
+
+#### Example 1: Barcode Scanner Integration { #capture_app_android_settings_webapp_custom_intents_examples_barcode }
+
+This example shows how to integrate a barcode scanner app to populate a patient ID field.
+
+**Configuration:**
+- **Intent Name**: Barcode Scanner for Patient ID
+- **Intent Description**: Scans patient ID barcodes using ZXing scanner
+- **Element Type**: Tracked Entity Attribute
+- **Attribute**: Patient ID
+- **Screen/Action**: DATA_ENTRY
+- **Package Name**: `com.google.zxing.client.android.SCAN`
+
+**Request Parameters:**
+| Key | Value |
+|-----|-------|
+| SCAN_MODE | 'QR_CODE_MODE' |
+| PROMPT_MESSAGE | 'Scan patient ID barcode' |
+
+**Response:**
+- **Extra Name**: `SCAN_RESULT`
+- **Extra Type**: String
+- **Property in JSON Object**: *(leave empty for string type)*
+
+#### Example 2: GPS Coordinates Integration { #capture_app_android_settings_webapp_custom_intents_examples_gps }
+
+This example demonstrates capturing GPS coordinates from a mapping application that returns JSON data.
+
+**Configuration:**
+- **Intent Name**: GPS Location Capture
+- **Intent Description**: Captures GPS coordinates from mapping app
+- **Element Type**: Data Element
+- **Data Element**: Household Location
+- **Screen/Action**: DATA_ENTRY
+- **Package Name**: `com.example.gpsapp.CAPTURE_LOCATION`
+
+**Request Parameters:**
+| Key | Value |
+|-----|-------|
+| ACCURACY | 'HIGH' |
+| TIMEOUT | 60000 |
+
+**Response:**
+- **Extra Name**: `LOCATION_DATA`
+- **Extra Type**: JSON Object
+- **Property in JSON Object**: `coordinates.latitude`
```

Comment:
Nested properties are not supported, are they? I am not sure about this @nancyespinoza @xavimolloy